### PR TITLE
CodeFactor cleanup

### PR DESCRIPTION
**Pull request by RDIL**
Signed-off-by: Reece Dunham <me@rdil.rocks>

# PR Summary

Trying to improve the codefactor score

because I have nothing better to do
ahhhhh

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [X] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [X] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [X] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [X] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs [hunk 1]
```diff
@@ -381,17 +381,19 @@ public void Cleanup()
...
         /// <param name="sessionId"></param>
         /// <param name="instanceId"></param>
         /// <param name="name"></param>
+        /// <param name="computerName"></param>
+        /// <param name="protocol"></param>
         /// <returns></returns>
         internal PSObject AddObjectToCache(
             CimSession session,
-            UInt32 sessionId,
+            uint sessionId,
```
Comment (TravisEz13) [inline src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs:396]: note to self, review change

File: src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs [hunk 2]
```diff
@@ -814,7 +818,7 @@ public CimSessionBase()
 
         /// <summary>
         /// <para>
-        /// Default runspace id
+        /// Default runspace ID.
```
Comment (TravisEz13) [inline src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs:821]:
```suggestion
        /// Default runspace Id.
```

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in an hour.

File: src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs (hunk 2 above)
Comment (TravisEz13) [reply src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs:821]: This is just a nit.  Not blocking.

**Comment by PoshChan** [comment]
@TravisEz13, I do not understand: Please remind me in an hour.

<details>
<summary>Commands available in this repo for you:
</summary>
<ul><li>
<code>retry &lt;target&gt;</code> this will attempt to retry only the failed jobs for the target pipeline, <code>restart</code> can be used in place of <code>retry</code>
</li>
<li>
<code>rebuild &lt;target&gt;</code> this will perform a complete rebuild of the target pipeline, <code>rerun</code> can be used in place of <code>rebuild</code> Supported values for &lt;target&gt; which can be a comma separated list are: <code>static</code>,<code>windows</code>,<code>macos</code>,<code>linux</code>,<code>all</code>
</li>
<li>
<code>get failures</code> this will attempt to get the latest failures for all of the target pipelines
</li>
<li>
<code>remind me in &lt;value&gt; &lt;units&gt;</code> this will create a reminder that will be posted after the specified duration &lt;value&gt; is a number, and &lt;units&gt; can be <code>minutes</code>, <code>hours</code>, or <code>days</code> (singular or plural)
</li>

</ul>
</details>

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
> Trying to improve the codefactor score
> 
> because I have nothing better to do
> ahhhhh

You could continue #11916 after it will be merged.

**Comment by RDIL** [comment]
If I have time, maybe

**Comment by iSazonov** [comment]
@TravisEz13 It seems the PR is ready to merge.

**Review by TravisEz13 — APPROVED** [review]

**Comment by iSazonov** [comment]
@RDIL Thanks!

**Pull request merged**